docs/remote/devcontainer-cli.md

```diff
@@ -1,220 +1,140 @@
+A consistent, predictable environment is key to a productive and enjoyable software development experience.
 
-## Visual Studio Code devcontainer CLI
+Containers (such as [Docker](https://www.docker.com)) have historically been used to standardize apps when they're deployed, but there's a great opportunity to support additional scenarios, including continuous integration (CI), test automation, and full-featured coding environments. A **development container** provides this working environment and ensures your project has the tools and software it needs, whether it's complex and distributed or just has a few requirements.
 
-Given the growing number of use cases for dev containers, there is a companion `devcontainer` CLI that can be used independent of the [Remote - Containers extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers) or GitHub Codespaces. This article will walk you through its installation and how to use it in different scenarios.
+![Diagram comparing dev versus production containers](images/devcontainer-cli/dev-container-stages.png)
 
-## System requirements
-
-To use the VS Code `devcontainer` CLI, you'll need the following on your system or CI/DevOps environment:
+Develop containers are supported in Visual Studio Code via the [Remote - Containers extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers) and in [GitHub Codespaces](https://docs.github.com/en/codespaces/setting-up-your-project-for-codespaces/introduction-to-dev-containers). This support is backed by [devcontainer.json](/docs/remote/devcontainerjson-reference.md), a structured JSON with Comments (jsonc) metadata format to configure a containerized environment.
 
-1. [Node.js 14+](https://nodejs.org).
-1. [The `docker` CLI](/docs/remote/containers#installation).
+As containerizing production workloads becomes commonplace, dev containers have become broadly useful for scenarios beyond VS Code. To promote dev containers, work has started on the [Development Containers Specification](https://github.com/devcontainers/spec), which empowers anyone in any tool to configure a consistent dev environment. As a companion to the specification, there is the open-source **dev container CLI**.
 
-## Installation
+## The dev container CLI
 
-### Install using VS Code
+The dev container CLI is a reference implementation for the dev container specification.
 
-1. Ensure you have the latest version of the [Remote - Containers extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers) installed (must be at least `v0.188.0`).
+When tools like VS Code and Codespaces detect a `devcontainer.json` file in a user's project, they use a CLI to configure a dev container. The devcontainer CLI is a reference implementation so that individual users and other tools can read in `devcontainer.json` metadata and create dev containers from it.
```

Comment:
```suggestion
When tools like VS Code and Codespaces detect a `devcontainer.json` file in a user's project, they use a CLI to configure a dev container. The dev container CLI is a reference implementation so that individual users and other tools can read in `devcontainer.json` metadata and create dev containers from it.
```